crates/rust-analyzer/src/main_loop.rs

```diff
@@ -424,13 +423,11 @@ impl GlobalState {
                 if let Ok(path) = from_proto::vfs_path(&params.text_document.uri) {
                     let DidChangeTextDocumentParams { text_document, content_changes } = params;
                     let vfs = &mut this.vfs.write().0;
-                    let world = this.snapshot();
                     let file_id = vfs.file_id(&path).unwrap();
 
                     // let file_id = vfs.file_id(&path).unwrap();
```

Comment:
Can you also remove this commented line (or use git revert)?

Reply:
Yes I can remove the commented line

Reply:
@lnicola is there a specific way to use `git revert` for this? I haven't used it before.